server/src/main/java/io/druid/server/QueryResource.java

```diff
@@ -151,6 +154,7 @@ public Response doPost(
         );
       }
 
+      Thread.currentThread().setName("qtp-query-" + queryId);
```

Comment:
what's the purpose of setting the thread name, is it just for logging or for some other debugging purposes?